diff --git a/foreign_cc/private/framework.bzl b/foreign_cc/private/framework.bzl
index 76192ef..acc0254 100644
--- a/foreign_cc/private/framework.bzl
+++ b/foreign_cc/private/framework.bzl
@@ -525,8 +525,6 @@ def cc_external_rule_impl(ctx, attrs):
     cc_toolchain = find_cpp_toolchain(ctx)
 
     execution_requirements = {tag: "" for tag in ctx.attr.tags}
-    if "requires-network" not in execution_requirements:
-        execution_requirements["block-network"] = ""
 
     # TODO: `additional_tools` is deprecated, remove.
     legacy_tools = ctx.files.additional_tools + ctx.files.tools_deps
